package.json

```diff
@@ -120,8 +119,7 @@
     "react-dom": "^16.8.0",
     "react-router-dom": "^5.2.0",
     "react-syntax-highlighter": "^12.2.1",
-    "react-test-renderer": "^16.13.1",
-    "regenerator-runtime": "^0.13.5",
```

Comment:
react-test-renderer should be in sync with react version. regenerator runtime was leftover from old docs site.